Tests/Fixtures/Entity/ContainerEntity.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Sonata\DoctrineORMAdminBundle\Tests\Fixtures\Entity;
+
+class ContainerEntity
```

Comment:
I looked for doc blocks in the VersionedEntity class since that had none I didn't specify one either.

What should the doc block specify for this fixture class?